functions/packages.sh

```diff
@@ -409,37 +409,51 @@ speedtest_cli_setup() {
 ## Function for installing and configure InfluxDB and Grafana while also integrate it to openHAB.
 ## The function can be invoked either INTERACTIVE with userinterface UNATTENDED.
 ##
-## When called UNATTENDED it will install both InfluxDB and Grafana on the local system. 
+## When called UNATTENDED it will install both InfluxDB and Grafana on the local system.
 ##
 ##    influxdb_grafana_setup()
 ##
 
 influxdb_grafana_setup() {
-  local FAILED=0
-  local text_intro="This will install and configure InfluxDB and Grafana. For more information please consult this discussion thread:
-  \nhttps://community.openhab.org/t/13761/1"
-  local text_fail="Sadly there was a problem setting up the selected option. Please report this problem in the openHAB community forum or as a openHABian GitHub issue."
-  local text_success="Setup successful. Please continue with the instructions you can find here:\n\nhttps://community.openhab.org/t/13761/1"
-
-  echo "$(timestamp) [openHABian] Setting up InfluxDB and Grafana... "
-  if [ -n "$INTERACTIVE" ]; then
-    if ! (whiptail --title "Description, Continue?" --yes-button "Continue" --no-button "Back" --yesno "$text_intro" 15 80) then echo "CANCELED"; return 0; fi
-  fi
-
-  local influxdb_address
-  local influxdb_admin_username
+  local FAILED
+  local text_intro text_fail text_success
+  local influxdb_address influxdb_admin_username
   local influxdb_admin_password
   local influxdb_openhab_username
   local influxdb_openhab_password
   local influxdb_grafana_username
   local influxdb_grafana_password
   local influxdb_database_name
   local grafana_admin_password
-  local openhab_integration=false
+  local openhab_integration
+  local text_influxDB_intro
+  local text_influxDB_configure
+  local influxdb_returncode
+  local text_influxdb_address
+  local text_influxDB_admin_password
+  local matched
+  local password_check
+  local text_grafana_admin_password
+  local text_openHAB_integration
+  local dist codename
+  local architecture
```

Comment:
@EliasGabrielsson @mstormi at which point did you guys decide to work with local? Could you please direct me to a discussion, if there is one? Thanks :)

Reply:
This commit doesn't change to code to use local.
It changes when local is assigned according to SC2155
( https://github.com/koalaman/shellcheck/wiki/SC2155 )
I simply took all instances where a local variable was defined and a value assigned immediately, and put the variable definitions c-style at the top of the function.

Reply:
@ThomDietrich See this issue: https://github.com/openhab/openhabian/issues/373
@GruensFroeschli The refactoring is good, we shall keep it. 👍

Reply:
> @EliasGabrielsson @mstormi at which point did you guys decide to work with local?

I am not aware of any discussion, and IIRC it was either Elias or Benjy to tell me.

Reply:
All fine by me. @GruensFroeschli you did good :) I was just curious.